torchgeo/datasets/spacenet.py

```diff
@@ -154,10 +165,12 @@ def _load_image(self, path: Path) -> tuple[Tensor, Affine, CRS]:
         Returns:
             the image
         """
-        filename = os.path.join(path)
-        with rio.open(filename) as img:
-            array = img.read().astype(np.int32)
-            tensor = torch.from_numpy(array).float()
+        with rio.open(path) as img:
```

Comment:
How common is hitting this? I'm assuming this is an edge cases for several patches per dataset

Reply:
See https://github.com/microsoft/torchgeo/pull/2203#issuecomment-2266679557